end_to_end/test_tflops_v4_8_one_slice.sh

```diff
@@ -17,7 +17,7 @@ fi
 #Setup and Train
 bash setup.sh
 python3 MaxText/train.py MaxText/configs/base.yml run_name=$RUN_NAME\
-    steps=150 reuse_example_batch=1 remat_policy='full' enable_checkpointing=False metrics_file='metrics.txt'\
+    steps=15 reuse_example_batch=1 remat_policy='full' enable_checkpointing=False metrics_file='metrics.txt'\
```

Comment:
I think you may have missed changing this back to 150 after testing

Reply:
Fixed.